### PR TITLE
Copyedit requests by PLOS

### PR DESCRIPTION
**Pull request by morgantaschuk**
- Updated the tex document to PLOS CB January 2017 version
- Removed second numbers from Rules (it was reading "1 Rule 1")
- Moved footnotes into references
- Reverted the changes @gvwilson made to incorporate the bbl file into the tex document so I could compile
- added a new command to make file : `make final` that creates a new tex document and puts the bbl file into it
- moved all the pdfs, posters, etc into the `dist` directory
- removed extra files from directory

The only thing that's unresolved is #54, because the paper still isn't in an official issue. But we can modify the ref and regenerate the tex quickly once it comes out.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,12 +12,24 @@ commands :
...
-	${LATEX} ${STEM}
-	${LATEX} ${STEM}
-	${LATEX} ${CHK}
-	${LATEX} ${CHK}
+	${LATEX} ${ROBU}
+	${BIBTEX} ${ROBU}
+	${LATEX} ${ROBU}
+	${LATEX} -output-directory=dist ${ROBU}
+	${LATEX} -jobname=S1_Checklist ${CHK}
+	${LATEX} -jobname=S1_Checklist -output-directory=dist ${CHK}
```
Comment (gvwilson) [inline Makefile:20]: Why these changes?

File: robust-checks.tex [hunk 2]
```diff
@@ -1,134 +1,29 @@
 \documentclass[10pt,letterpaper]{article}
-\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}
-
-% amsmath and amssymb packages, useful for mathematical formulas and symbols
-\usepackage{amsmath,amssymb}
-
-% Use adjustwidth environment to exceed column width (see example table in text)
-\usepackage{changepage}
+\usepackage[top=0.85in,left=1.0in,footskip=0.75in]{geometry}
```
Comment (gvwilson) [inline robust-checks.tex:2]: Why the change in geometry? Is this from PLOS?

File: robust-checks.tex [hunk 3]
```diff
@@ -1,134 +1,29 @@
...
-{\ddag} These authors contributed equally to this work.
-\\
-* Corresponding author.
 \end{flushleft}
 
 
 \begin{easylist}[checklist]
 & Use version control.
-&& Put everything created manually into version control as soon as it is created.
+&& Put everything into version control as soon as it is created.
```
Comment (gvwilson) [inline robust-checks.tex]: Why remove "created manually"? We don't put e.g. output files from compilation in version control.

**Review by gvwilson — COMMENTED** [review]
A couple of questions, and one comment about removing "by hand" from the version control rule - happy to have that change in if you want.

File: Makefile (hunk 1 above)
Comment (morgantaschuk) [reply Makefile:20]: It changes the name of the output file to S1_checklist, according to PLOS submission guidelines, and puts the final documents into the dist folder.

File: robust-checks.tex (hunk 2 above)
Comment (morgantaschuk) [reply robust-checks.tex:2]: No, this is for the checklist. It doesn't need to be in PLOS format, since it's supplemental. I tweaked it until it looked good and everything fit on one page.

File: robust-checks.tex (hunk 3 above)
Comment (morgantaschuk) [reply robust-checks.tex]: Most likely I changed this so it would fit onto a single line. :) I can put it back.

**Comment by gvwilson** [comment]
As-is is fine.

File: robust-checks.tex (hunk 3 above)
Comment (morgantaschuk) [reply robust-checks.tex]: oh, now I remember. It was because this line read "created manually... as soon as it is created".

**Comment by gvwilson** [comment]
Ah - I agree, take it out.

**Comment by morgantaschuk** [comment]
Your point is still a good one, since we don't put compiled things in VC. (except in this project..)

**Comment by morgantaschuk** [comment]
"Put everything that you write into version control as soon as it is created."?

**Comment by gvwilson** [comment]
:+1:
